src/coreclr/src/vm/typedesc.cpp

```diff
@@ -40,8 +40,7 @@ BOOL ParamTypeDesc::Verify() {
 
     _ASSERTE(m_TemplateMT.IsNull() || GetTemplateMethodTableInternal()->SanityCheck());
     _ASSERTE(!GetTypeParam().IsNull());
-    BAD_FORMAT_NOTHROW_ASSERT(GetTypeParam().IsTypeDesc() || !GetTypeParam().AsMethodTable()->IsArray());
```

Comment:
`BAD_FORMAT_NOTHROW_ASSERT` fires only when you are running with debugger attached. I have seen this fire for `ref int[]` under debugger. This assert assumed that arrays have ParamTypeDescs.

Reply:
Looks like I missed this one. Thanks!

---
In reply to: [371405798](https://github.com/dotnet/runtime/pull/2237#discussion_r371405798) [](ancestors = 371405798)